forui/example/pubspec.yaml

```diff
@@ -29,6 +29,7 @@ dependencies:
     path: ../
   intl: ^0.19.0
   sugar: ^3.1.0
+  wakelock_plus: ^1.2.8
```

Comment:
I don't think this should be a dependency. It should either be a `dev_dependency` or we remove it entirely.

Reply:
Nope it can't, but I think it's fine since it's the example's pubspec.